server/conf/application.conf

```diff
@@ -52,6 +52,11 @@ whitelabel {
   # The full display name of the civic entity, will use "City of Seattle" if not set.
   civic_entity_full_name = "City of Seattle"
   civic_entity_full_name = ${?WHITELABEL_CIVIC_ENTITY_FULL_NAME}
+
+  # The URL of a 32x32 or 16x16 pixel [favicon](https://developer.mozilla.org/en-US/docs/Glossary/Favicon)
+  # image, in GIF, PNG, or ICO format.
```

Comment:
Looks like we only have route entries for "ico" / "png". Please either remove "gif" from here or add it as a supported route.

Reply:
That's not really needed - there's 2 things going on here:
1) we set the rel=icon link metadata in the head to the image you specify (the main way to set the favicon)
2) we redirect any browsers who, by default, request favicon from root, to the correct image link. (it's default behaviour, but doesn't follow the spec - https://stackoverflow.com/questions/56222166/prevent-browser-from-trying-to-load-favicon-from-root-directory)

So those routes entries only are needed for the routes that a browser may request, but we can actually return an image in any requested format

Reply:
Added comments